websocket/connection.go

```diff
@@ -210,17 +220,22 @@ func (c *ManagedConnection) Send(msg interface{}) error {
 		return err
```

Comment:
I think it is better to encode outside the lock.
To hold lock for minimum possible time.